08-components-lab/src/01-timeconverter.jsx

```diff
@@ -1,12 +1,50 @@
-/**
- * Lab 01: 
- *
- * Implement a time converter component.
- * Component should show 3 input fields labeled hours, minutes and seconds.
- * Typing a value in either field updates the other two with corresponding values.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+/**
+ * Lab 01: 
+ *
+ * Implement a time converter component.
+ * Component should show 3 input fields labeled hours, minutes and seconds.
+ * Typing a value in either field updates the other two with corresponding values.
+ */
+
+import React from 'react';
+
+export default class TimeConverter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            number: 0,
+        };
+    }
+
+    refresh = (num) => {

```

Comment:
I mean that instead of doing
```
refresh = (num) => {

}
```
which will show anonymous function in stack trace when debugging,
you can bind method in constructor in the following way.
the method:
```
refresh(){
}
```
constructor:
```
constructor(props){
 //...
this.refresh = this.refresh.bind(this);
 
}
```